data/org.eclipse.birt.report.data.oda.jdbc/src/org/eclipse/birt/report/data/oda/jdbc/JDBCDriverManager.java

```diff
@@ -70,8 +71,17 @@ public class JDBCDriverManager {
 	public static final String JDBC_PASSWORD_PROP_NAME = "password"; //$NON-NLS-1$
 
 	private static final String INVALID_AUTH_SQL_STATE = "28000"; // X/Open SQL State //$NON-NLS-1$
+
 	private static final String EMPTY_STRING = ""; //$NON-NLS-1$
 
+	private boolean isDriverPropertyValidationEnabled = true;
+
+	private String excludedDriverForValidation;
```

Comment:
The property name "...excluded.drivers" suggests multiple exclusions are support.  This field suggest a single one is specified.  Later it just does a contains test on the String, so the separator doesn't matter.   If you were to write some documentation somewhere how to use system property to exclude multiple drivers from validation, how would you describe the syntax for it?

Reply:
Yes, I will create a documentation of it with a README.MD.
Yes, the delimitor is independend because the class name will be checked with "contain()" but I recommand ","-comma.